ctgan/synthesizers/tvae.py

```diff
@@ -176,6 +178,12 @@ def fit(self, train_data, discrete_columns=()):
                 optimizerAE.step()
                 self.decoder.sigma.data.clamp_(0.01, 1.0)
 
+            if self.verbose:
+                print(f'Epoch {i+1}, Loss: {loss.detach().cpu(): .4f},',  # noqa: T001
+                      f' Rec loss: {loss_1.detach().cpu(): .4f},',
+                      f' KL loss: {loss_2.detach().cpu(): .4f}',
+                      flush=True)
```

Comment:
would you mind adding unit tests for this case? Just making sure the print out only happens when verbose is true and that the output is as expected

Reply:
Sure. Are there any active unit tests run for TVAE as a template? I see `test_tvae.py` but it looks like only placeholder comments.

Reply:
Unfortunately I don't think there are, although I think a PR with some should be coming in shortly. I would use the tests in this file as an [example](https://github.com/sdv-dev/CTGAN/blob/master/tests/unit/synthesizer/test_ctgan.py). I know the tests don't exist for the CTGAN verbose parameter, but we're trying to improve test coverage.

I think you can do something similar to [this](https://github.com/sdv-dev/SDV/blob/067c4e3e40631083a283e9bf8cafa525228ce9b2/tests/unit/multi_table/test_base.py#L86-L96). Mock print and make sure it gets called with the correct string but only if verbose is True.